e2e-tests/portal/cypress/integration/journeys/dashboard/dashboard-deals/dashboard-deals-create-deal.spec.js

```diff
@@ -0,0 +1,49 @@
+const relative = require('../../../relativeURL');
+const mockUsers = require('../../../../fixtures/mockUsers');
+
+const { dashboard, selectScheme } = require('../../../pages');
+
+const BANK3_GEF_MAKER1 = mockUsers.find((user) => (user.roles.includes('maker') && user.username === 'BANK3_GEF_MAKER1'));
+const BANK1_MAKER1 = mockUsers.find((user) => (user.roles.includes('maker')));
+
+context('Create application as gef-only bank and other bank', () => {
+  describe('Create application as gef-only bank', () => {
+    before(() => {
+      Cypress.Cookies.preserveOnce('connect.sid');
+      cy.login(BANK3_GEF_MAKER1);
+      dashboard.visit();
+      cy.url().should('eq', relative('/dashboard/deals/0'));
+    });
+
+    it('When clicking Create new button, it takes user straight to mandatory criteria page', () => {
+      dashboard.createNewSubmission().click();
+      cy.url().should('eq', relative('/gef/mandatory-criteria'));
+    });
+
+    it('When clicking continue, it should lead to name your application', () => {
+      dashboard.mandatoryCriteriaYes().click();
+      dashboard.continueButton().click();
+      cy.url().should('eq', relative('/gef/name-application'));
+    });
+  });
+
```

Comment:
Can we add a test for manually navigating by URL to some BSS "create/start a deal" pages, as a GEF only bank?

Reply:
Ignore - ideally we'd have middleware to cover all BSS routes. This is not a small job and we can't do right now - as it is now is ok for V1.